features/commands.js

```diff
@@ -85,6 +85,32 @@ https://reactjs.org/blog/2018/06/07/you-probably-dont-need-derived-state.html`,
         });
       }
     },
+    {
+      words: [`!stateupdates`, `!su`],
+      help: `Explains the issue with state updates being asynchronous`,
```

Comment:
there isn't really an issue with state updates being asynchronous, so I don't think `issue` is the right wording here.
```suggestion
      help: `Explains the implications involved with state updates being asynchronous`,
```
I'm not sure if this is any better? 🤔